src/components/TempTodo/TempTodo.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  tempTodo: Todo;
+  isLoading: boolean;
+};
+
+export const TempTodo: React.FC<Props> = ({ tempTodo, isLoading }) => {
```

Comment:
You can just name the prop `todo`. It's definitely a component of the Todo, so there is no need to name `tempTodo` this prop. It's about naming only. It's not an error.